ComponentKit/TransactionalDataSources/Common/CKTransactionalComponentDataSourceAppliedChanges.mm

```diff
@@ -48,6 +48,11 @@ - (instancetype)initWithUpdatedIndexPaths:(NSSet *)updatedIndexPaths
   return self;
 }
 
+- (NSString *)description
+{
+  return [NSString stringWithFormat:@"Updated Index Paths: %@\nRemoved Index Paths: %@\nRemove Sections: %@\nMoves: %@\nInserted Sections: %@\nInserted Index Paths: %@", _updatedIndexPaths, _removedIndexPaths, _removedSections, _movedIndexPaths, _insertedSections, _insertedIndexPaths];
```

Comment:
split on a new line. also it would be nice to use a more standard syntax for description; this isn't very common